app/javascript/src/ChangePassword/components/ChangePassword.jsx

```diff
@@ -0,0 +1,92 @@
+// @flow
+
+import React, { useState } from 'react'
+import { createReactWrapper } from 'utilities/createReactWrapper'
+import { LoginPage, Form, ActionGroup, Button } from '@patternfly/react-core'
+import { HiddenInputs, FlashMessages } from 'LoginPage'
+import { PasswordInput } from 'ChangePassword'
+import type { FlashMessage } from 'Types'
+import { validateForm } from 'LoginPage/utils/formValidation'
+
+import 'LoginPage/assets/styles/loginPage.scss'
+import brandImg from 'LoginPage/assets/images/3scale_Logo_Reverse.png'
+import PF4DownstreamBG from 'LoginPage/assets/images/PF4DownstreamBG.svg'
+
+type Props = {
+  lostPasswordToken: ?string,
+  url: string,
+  errors: (?FlashMessage)[]
+}
+
+const validationConstraints = {
+  'user[password]': {
+    presence: true,
+    length: { minimum: 1 }
+  },
+  'user[password_confirmation]': {
+    presence: true,
+    length: { minimum: 1 }
+  }
+}
+
+const ChangePassword = ({ lostPasswordToken, url, errors }: Props) => {
```

Comment:
This I'll let it for another PR, but the `ChangePassword` component could be agnostic and independent component. Meaning we can reuse it in different parts of the app, like the ones in  https://github.com/3scale/porta/tree/master/app/javascript/src/Form